<launch>
    <arg name="x" default="1.8"/> <!-- set to starting location for the scenario in map frame-->
    <arg name="y" default="2.8"/>
    <arg name="theta" default="1.5708"/>

    <include file="$(find roah_rsbb_comm_ros)/launch/test.launch"/>

    <node name="rsbb_node" pkg="fake_rsbb" type="rsbb.py" output="screen">
        <param name="benchmark" value="restricted_task_3"/>

        <!-- summon location -->
        <param name="x" value="$(arg x)"/>
        <param name="y" value="$(arg y)"/>
        <param name="theta" value="$(arg theta)"/>

        <!-- initial box pose TODO inherit this -->
        <param name="box_initial_x" value="-2.7"/>
        <param name="box_initial_y" value="1.0"/>
        <param name="box_initial_z" value="1.83"/>

        <!-- final box pose TODO inherit this -->
        <param name="box_final_x" value="-2.7"/>
        <param name="box_final_y" value="-3.82"/>
        <param name="box_final_z" value="1.31"/>

    </node>

</launch>